website/src/components/common/ActiveFilters.tsx

```diff
@@ -4,25 +4,41 @@ import type { SequenceFilter } from '../SearchPage/DownloadDialog/SequenceFilter
 
 type ActiveFiltersProps = {
     sequenceFilter: SequenceFilter;
+    removeFilter?: (key: string) => void;
 };
 
-export const ActiveFilters: FC<ActiveFiltersProps> = ({ sequenceFilter: downloadParameters }) => {
-    if (downloadParameters.isEmpty()) return null;
+export const ActiveFilters: FC<ActiveFiltersProps> = ({ sequenceFilter, removeFilter }) => {
+    if (sequenceFilter.isEmpty()) return null;
+    const showXButton = removeFilter !== undefined;
 
     return (
-        <div className='mb-4'>
-            <h4 className='font-bold mb-2'>Active filters</h4>
-            <div className='flex flex-row flex-wrap gap-3'>
-                {[...downloadParameters.toDisplayStrings()].map(([key, [label, value]]) => (
-                    <div
-                        key={key}
-                        className='border-primary-600 rounded-sm border border-l-primary-600 bg-gray-100 border-l-8 px-3 py-1 text-sm'
-                    >
-                        <span className='text-primary-900 font-light pr-1'>{label}:</span>
-                        <span className='text-primary-900 font-semibold'>{value}</span>
-                    </div>
-                ))}
-            </div>
+        <div className='flex flex-row flex-wrap gap-3'>
+            {[...sequenceFilter.toDisplayStrings()].map(([key, [label, value]]) => (
+                <div
+                    key={key}
+                    className='border-primary-600 rounded-sm border border-l-primary-600 bg-gray-100 border-l-8 pl-3 py-1 text-sm flex flex-row'
+                >
+                    <span className='text-primary-900 font-light pr-1'>{label}:</span>
+                    <span className='text-primary-900 font-semibold'>{value}</span>
+                    {showXButton ? (
+                        <button
+                            aria-label='remove filter'
+                            className='inline ml-2 mt-0.5 pr-2'
+                            onClick={() => removeFilter(key)}
+                        >
+                            <svg className='w-3 h-4 text-primary-600' fill='currentColor' viewBox='0 0 20 20'>
```

Comment:
We should refactor this sometime but I know it's in mine, let's leave it for now 👍